Adding links to related pages (#2681)
elastic/docs-content: manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover.md

3. Move the index into the `delete` phase 90 days after rollover.
4. Trigger the `delete` action when the index enters the delete phase.

:::
::::

:::{tip}
For more details about default {{ilm-init}} policy settings, refer to [Create a lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md#ilm-create-policy).
:::

### Create an index template to create the data stream and apply the lifecycle policy [ilm-gs-apply-policy]

To set up a data stream, first create an index template to specify the lifecycle policy. Because the template is for a data stream, it must also include a `data_stream` definition.

> For example, you might create a template named `timeseries_template` and use for a future data stream named `timeseries`.

Reviewer: ```suggestion
For example, you might create a template named `timeseries_template` and use it for a future data stream named `timeseries`.
```

To enable {{ilm-init}} to manage the data stream, the template configures one {{ilm-init}} setting:

* `index.lifecycle.name` specifies the name of the lifecycle policy that you want to apply to the data stream.

> You can use the {{kib}} **Create template** wizard to add the template or the [Create or update index template](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-index-template) API to add an index template to a cluster and apply the lifecycle policy to indices matching the template.

Reviewer: If this is accurate, it might be good to make this sentence slightly shorter
```suggestion
Use the {{kib}} **Create template** wizard to add a template or the [Create or update index template](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-index-template) API to add a template and apply the lifecycle policy to indices matching the template.
```

::::{tab-set}
:group: kibana-api
:::{tab-item} {{kib}}
:sync: kibana

> To add an index template to a cluster, go to **Stack Management > Index Management**. In the **Index Templates** tab, click **Create template**.

Author: Fixing all the comments in a separate commit. Thank you!

![Create template page](/manage-data/images/elasticsearch-reference-create-index-template.png "")

This wizard invokes the create or update index template API to create the index template with the options you specify.

:::{tip} 

> For more information about the available index template options that you can specify, refer to [Create an index template to apply the lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md#apply-policy-template).

Reviewer: ```suggestion
To learn about which index template options you can specify, refer to [Create an index template to apply the lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md#apply-policy-template).
```

### Check lifecycle progress [ilm-gs-check-progress]

> To get status information for managed indices, you can use {{kib}} to [view their current status, including details about the ILM policy](/manage-data/lifecycle/index-lifecycle-management/policy-view-status.md) or the {{ilm-init}} explain API. This lets you find out things like:

Reviewer: I'm not sure if this is accurate, but suggesting a way to make the sentence simpler

```suggestion
Use {{kib}} to [view the current status of your managed indices](/manage-data/lifecycle/index-lifecycle-management/policy-view-status.md), including details about the ILM policy or the {{ilm-init}} explain API. Find out things like:
```

* `index.lifecycle.name` specifies the name of the lifecycle policy to apply to new indices that match the index pattern.
* `index.lifecycle.rollover_alias` specifies the index alias to be rolled over when the rollover action is triggered for an index.

> ::::{tab-set}

Reviewer: You could consider using a [snippet](https://elastic.github.io/docs-builder/syntax/file_inclusion/) if the content of this tab is the same as the one above/elsewhere on the page. It'll be easier to keep the info consistent
